controllers/contacts.js

```diff
@@ -4,7 +4,13 @@ const { schemas } = require("../models/contact");
 
 const getAllContacts = async (req, res, next) => {
   try {
-    const result = await Contact.find();
+    const { _id: owner } = req.user;
+    const { page = 1, limit = 20 } = req.query;
```

Comment:
add favorite = true to const in 8th line
const { page = 1, limit = 20, favorite = true } = req.query;